examples/segnet/train.py

```diff
@@ -0,0 +1,104 @@
+import matplotlib  # isort:skip # NOQA
+matplotlib.use('Agg')  # isort:skiip # NOQA
+
+import argparse
+import time
+
+import chainer
+import numpy as np
+
+from chainer import iterators
+from chainer import optimizers
+from chainer import training
+from chainer.training import extensions
+from chainercv.datasets import CamVidDataset
+from chainercv.datasets import TransformDataset
+from chainercv.links.loss import PixelwiseSoftmaxLossWithWeight
+from chainercv.links.model.segnet import segnet_basic
+
+
```

Comment:
@yuyu2172 I just tried to make the training script as simple as possible. Because it doesn't need to change many parameters by giving args to the script because its goal is to replicate the paper's result. Is it unnatural?